src/operations/lib/validations/phone_number.ts

```diff
@@ -0,0 +1,17 @@
+import { DataEntity } from '@terascope/job-components';
+import _ from 'lodash';
+import PhoneValidator from 'awesome-phonenumber';
+import OperationBase from '../base';
+import { OperationConfig } from '../../../interfaces';
+
+export default class PhoneNumber extends OperationBase {
```

Comment:
This is supposed to be called `isdn`.

https://github.com/terascope/ts-transforms/issues/30